packages/ai/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@kof is this alright?

Reply:
I would probably keep it aligned with the rest - agpl or is there a good reason for MIT?